scripts/process.py

```diff
@@ -1,32 +1,40 @@
 #!/usr/bin/python
 # -*- coding: utf8 -*-
 
-import urllib
 import os
-import datetime
 import csv
 import xlrd
+import json
+import datetime
+import requests
 
 def setup():
-	'''Crates the directorie for archive if they don't exist
+	'''
+        Crates the directorie for archive if they don't exist
```

Comment:
@gradedSystem could you please fix this French to English, please? :smile:

Reply:
Yeah, resolving that